packages/components/src/Base/FC__Menu.re

```diff
@@ -16,14 +14,210 @@ external rcDividerClass: ReasonReact.reactClass = "Divider";
 
 [%bs.raw {|require("rc-menu/assets/index.css")|}];
 
+module Styles = {
+  open Css;
+  let menuBorderRadius = `px(3);
+  // Menu general, applied to menu <ul> elements
+  global(
+    ".ft-menu-general, .ft-submenu-general>ul",
+    [
+      fontFamily("Lato"),
```

Comment:
Please move "Lato" to the colors file or another config, and then reference it here and in other places. We shouldn't have it in so many places. We should only have this string in one place.